docker/production/docker-compose.yml

```diff
@@ -20,12 +20,14 @@ services:
     volumes:
       - /etc/localtime:/etc/localtime:ro
       ## Keep configs here.
-      - ./config:/root/.stash
+      - /your/path/to/Stash/config:/root/.stash
       ## Point this at your collection.
-      - ./data:/data
+      - /your/path/to/your/files:/data
       ## This is where pre-generated transcodes live.
-      - ./transcodes:/transcodes
+      - /your/path/to/Stash/transcodes:/transcodes
       ## This is where your stash's metadata lives
-      - ./metadata:/metadata
+      - /your/path/to/Stash/metadata:/metadata
       ## Any other cache content.
-      - ./cache:/cache
+      - /your/path/to/Stash/cache:/cache
+      ## Generated
```

Comment:
maybe 
## Where to store generated content (screenshots,previews,...)

Reply:
Same thing here as my previous